### DISCUSSION TITLE
Create/Install OS cluster using Helm

### DISCUSSION THREAD

**Discussion by IshaGirdhar** [Opster/opensearch-k8s-operator#448] [Ideas]
Currently, with Opster, we deploy the operator using Helm, but we use Kubectl to deploy an OS cluster. This is to propose providing an option of creating OS Cluster installations using Helm as well.

**Comment by neeraj-n-singh** [comment]
I am also been looking at the exact requirement for a long. This enhancement would be beneficial and help my company to easily adapt.

**Comment by rahulsrivastava71** [comment]
i am running on my env and facing the same issue , please help with fix for this
